extensions/vscode/lsp/.eslintignore

```diff
@@ -0,0 +1,5 @@
+node_modules/**
+client/node_modules/**
+client/out/**
+server/node_modules/**
+server/out/**
```

Comment:
I believe server/ no longer exists, so this should be removed.

Reply:
I agree.